pdns/dnspacket.hh

```diff
@@ -61,8 +61,9 @@ public:
   const string& getString(); //!< for serialization - just passes the whole packet
 
   // address & socket manipulation
-  void setRemote(const ComboAddress*);
+  void setRemote(const ComboAddress*, const ComboAddress* = nullptr);
```

Comment:
Just a nit but perhaps void `setRemote(const ComboAddress&, std::optional<ComboAddress> = boost::none);` would convey the intent a bit better?

Reply:
Yes! My compiler did not like `boost::none` but `std::nullopt` appears to work.